src/Nest/XPack/Slm/SnapshotLifecyclePolicyMetadata.cs

```diff
@@ -52,6 +52,31 @@ public class SnapshotLifecyclePolicyMetadata
 		/// </summary>
 		[DataMember(Name = "in_progress")]
 		public LifecycleSnapshotInProgress InProgress { get; internal set; }
+
+		/// <summary>
+		///	 Information about the last time the policy successfully initiated a snapshot.
+		/// </summary>
+		[DataMember(Name = "last_success")]
+		public SnapshotInvocationRecord LastSuccess { get; set; }
+
+		/// <summary>
+		///	 Information about the last time the policy failed to initiate a snapshot
+		/// </summary>
+		[DataMember(Name = "last_failure")]
+		public SnapshotInvocationRecord LastFailure { get; set; }
+	}
+
+	public class SnapshotInvocationRecord
```

Comment:
rename this to `SnapshotLifecycleInvocationRecord` in line with other SLM type names? The Java source uses `SnapshotInvocationRecord`, but has namespaces to reuse type names across the codebase.